pkg/live/fake-loader.go

```diff
@@ -0,0 +1,61 @@
+package live
```

Comment:
I think we can keep this one too in the cmdstatus package. It might be that we want to reuse it in the other kpt live commands, but we can move it if that happens.